README.md

```diff
@@ -212,69 +219,123 @@ module.exports = {
 | microsoftedge | 17.17134 | requires [Sauce Labs](https://screener.io/v2/docs/sauce) Integration |
 | safari | 11.1 | requires [Sauce Labs](https://screener.io/v2/docs/sauce) Integration |
 
+### Sauce Connect Integration
+
+When using Sauce Labs browsers, you have the option to use the Sauce Connect tunnel by setting the flag `launchSauceConnect: true`. When enabled, Sauce Connect will be launched and managed by this module, and assigned a unique tunnel identifier.
 
+  ```javascript
+  sauce: {
+    username: 'sauce_user',
+    accessKey: 'sauce_access_key',
+    maxConcurrent: 10, // optional available concurrency you have from Sauce Labs
+    launchSauceConnect: true // optional,
+  }
+  ```
 
+#### Important Notes on Sauce Connect
 
+- Sauce Connect Integration requires all browsers to be Sauce Labs Browsers. An error is thrown when using non-Sauce browsers.
 
-### Additional Configuration Options
+  If you don't follow the rules, the runner will throw you the error: `Only Sauce Labs browsers with version can be used when launchSauceConnect flag is enabled`.
+
+- Logs for Sauce Connect are saved in the root of your project under `sauce-connect.log` for debugging purposes.
+
+  The sauce connect tunnel will generate `sauce-connect.log` log file for debugging purpose at the root directory of your project.
+
+- Sauce Connect tunnel cannot be used together with the `tunnel` option.
+
+  `ngrok` tunnel is forbidden when you run the sauce connect tunnel, which means you can not set
```

Comment:
Please remove this and below. Just keep first line of bullet. Clients do not know what "ngrok" is